### ISSUE TITLE
Job killed when training supervised task with pretrained vectors

### ISSUE THREAD

**Issue by zampierimatteo91** [facebookresearch/fastText#965]
Hi,

First of all compliments for this amazing library and the code that stands below it.

Now I come to the issue:
I'm training supervised multi-class and multi-label models on a training set plus pretrained vectors.
One corpus of pretrained vectors is built on a text corpus I assembled, and is 365398035 bytes in size. Training the model with a learning rate of 1e-5/6, 25/50 epochs, dim 300, wordNgram 3, and default threads gives no issue.

When I try to train the model on the downloaded wiki vectors (4521473623 bytes), I get an error even with a learning rate some order of magnitudes smaller.

The command I issue is:
```
./fasttext supervised -input ../data/classification/fasttext/fasttext_multilabel.train -output model_mlabel_50_skip_itavec -wordNgrams 3 -bucket 200000 -lr 0.0000001 -dim 300 -epoch 25 -loss one-vs-all -saveOutput -seed 123 -pretrainedVectors ../data/classification/fasttext/italian_vectors.vec

```

And the output is:
```
Read 0M words
Number of words:  2842
Number of labels: 17
Killed
```

The `.train` file is not big, just ~300 documents with ~200 tokens per document.

I am working on a cluster, 1 node with 120GB ram.

Cheers

**Comment by SystemZ** [comment]
> Killed

This probably means that you ran out of RAM.
Watch `htop` or similar tool when you are launching `./fasttext`
